handbook/company/leadership.md

```diff
@@ -380,6 +380,12 @@ Once the new team member replies and accepts their offer in writing, 🔦 Head o
 Now what happens?  🔦 Business Operations will then follow the steps in the "Hiring" issue, which includes reaching out to the new team member within 1 business day from a separate email thread to get additional information as needed, prepare their agreement, add them to the company's payroll system, and get their new laptop and hardware security keys ordered so that everything is ready for them to start on their first day.
 
 
+## Creating 30-60-90 day plans 
+Before a new hire starts at Fleet, managers create a 30-60-90 day plan outlining key role objectives. These plans are reviewed weekly in 1:1 meetings during the first three months of employment, ensuring continuous support and alignment with company goals. 
```

Comment:
Let's pull this into the hiring steps than a separate section, that way we can ensure it happens.

Reply:
@Sampfluger88 we've updated the hiring/onboarding issue templates in this linked (confidential) PR to ensure this happens: https://github.com/fleetdm/confidential/pull/7503. 

I've pulled it up into the hiring section, but haven't added it to existing steps. Imo it doesn't fit into the steps prior to an offer being made and the hiring steps taken after an offer is made currently outline the responsibilities of biz ops. The formulation of a 30-60-90 would be undertaken by the manager. Felt like this was the minimum viable change given the current structure of the page and process. Wdyt?

Reply:
Let's try it as the new "4.". Archiving the Slack channel becoming "5." Changing it to match the others. 
e.g. `** Create a 30-60-90 day plan**:`

Happy to have a quick call re what I'm thinking also.

Reply:
Updated, will request new review.